eng/common/scripts/Create-APIReview.ps1

```diff
@@ -166,7 +166,8 @@ function IsApiviewStatusCheckRequired($packageInfo)
         return &$IsApiviewStatusCheckRequiredFn $packageInfo
     }
 
-    if ($packageInfo.SdkType -eq "client" -and $packageInfo.IsNewSdk) {
+    # API review is required for both client and management plane packages
+    if (($packageInfo.SdkType -eq "client" -or $packageInfo.SdkType -eq "mgmt") -and $packageInfo.IsNewSdk) {
```

Comment:
This will still be an issue. We don't need API view approval for GA release. What we need to block is the preview release only for mgmt plane when namespace is not approved. Current change will block GA release if entire API review is not approved for mgmt.

Reply:
You should add a separate method for namespace check required and that should be called from https://github.com/Azure/azure-sdk-tools/pull/11605/files#diff-ee60aabbce995095134759c343bf23ef121e2ff90c9ab61f38051d7f989f86b3L255 to avoid blocking mgmt GA release.